File: Java/count_digit.java
import java.util.*;
public class count_digit{
    public static void main(String[] args) {
        try(Scanner sc=new Scanner(System.in)){
            System.out.print("Enter the digit: ");
            int a=sc.nextInt(),c=0;
            while(a>0){
                a/=10;
                c++;
            }
            System.out.print("Count: "+c);
            sc.close();
        }
    }
}

File: C++/count_digit.cpp
#include<iostream>
using namespace std;
int main(){
    long long int a,r=0,c=0;
    cout<<"Enter the digit: ";
    cin>>a;
    while(a>0){
        a/=10;
        c++;
    }
    cout<<"Count of the digit is:"<<c;
    return 0;
}